airsonic-main/src/main/resources/liquibase/6.5/add-multiple-genres-join.xml

```diff
@@ -0,0 +1,88 @@
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd">
+    <!-- Drop the previous genre table and create a new one w/ an id primary index -->
+    <changeSet id="recreate-genre-table" author="devonbryant">
+        <preConditions onFail="MARK_RAN">
+            <not>
+                <columnExists tableName="genre" columnName="id"/>
+            </not>
+        </preConditions>
+        <dropTable tableName="genre"/>
```

Comment:
There is a drop table here. What about existing datas ?

Reply:
I think you have to drop the genre table otherwise you'd end up with this problem which is also a problem if you re-add a file library: https://imgur.com/V6LmjhE

Reply:
Very excited to see progress being made here! :)

Reply:
> 
> 
> @adylise What do you mean ?

Mult-genre support is something I've been really looking forward to eventually having, and I'm very happy to see some people working on it and testing stuff out for it. :)